guides/web-apps/acceptance-testing-with-selenium/README.md

```diff
@@ -114,114 +84,104 @@ We don't want to test only Firefox. Because of that, you should use Selenium Rem
-                InternetExplorerOptions options = new InternetExplorerOptions();
-                options.IgnoreZoomLevel = true;
-                capabilities = (DesiredCapabilities)options.ToCapabilities();
-                driver = new RemoteWebDriver(remoteWebDriverUri, capabilities, TimeSpan.FromSeconds(10));
-                break;
-            case "edge":
-                capabilities = DesiredCapabilities.Edge();
-                driver = new RemoteWebDriver(remoteWebDriverUri, capabilities);
-                break;
-            default:
-                capabilities = DesiredCapabilities.Firefox();
-                driver = new RemoteWebDriver(remoteWebDriverUri, capabilities);
-                break;
+            _browser = browser;
         }
-        return driver;
-    }
-}
-```
-
-Now, we need to adapt the test file to make use of it.
-
-In the file you created in the previous step (`PageLoads.cs`), locate the following line:
-
-```cs
-driver = new FirefoxDriver();
-```
-
-Change it to:
-
-```cs
-driver = WebDriverFactory.Create(this.browser);
-```
-
-Now, what is `this.browser`? This is a fixture parameter that carries the name of the browser used to execute the test.
 
-What this means is that you need to change every instance of :
+        [OneTimeSetUp]
+        public void TestFixtureSetUp()
+        {
+            if (_browsersTc != null)
+            {
+                _browsersToRun = _browsersTc.Split(',').ToList();
+            }
+            else
+            {
+                _browsersToRun.Add("Chrome");
+                _browsersToRun.Add("Firefox");
+                //_browsersToRun.Add("Edge");
+            }
+
+            if (_browsersToRun.Contains(Config.BrowserDictionary[_browser]))
+                Driver = WebDriverManager.StartDriver(_browser, Config.Timeout, Config.RemoteWebDriverUri);
+            else
+            {
+                Assert.Ignore(Config.BrowserDictionary[_browser] + " is on browsers ignore list");
+            }
+        }
 
-```cs
-[TestFixture]
-```
+        [OneTimeTearDown]
+        public void TestFixtureTearDown()
+        {
+            Driver?.Quit();
+        }
 
-To be:
+        protected TResult WaitUntil<TResult>(Func<IWebDriver, TResult> condition)
+        {
+            WebDriverWait wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(10));
+            return wait.Until(condition);
+        }
 
-```cs
-[TestFixture("firefox")]
-[TestFixture("chrome")]
-[TestFixture("edge")]
+        public bool WaitForText(IWebElement elementName, string text, int seconds)
+        {
+            WebDriverWait wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(seconds));
+            return wait.Until(ExpectedConditions.TextToBePresentInElement(elementName, text));
+        }
+    }
+}
 ```
 
-Finally, add this constructor somewhere on top of the class. It stores the fixture parameter in the `this.browser` variable:
-
+Constructor for every test class should be like this:
```

Comment:
Here I agree with @Mackiovello that it would be much more readable to just give two links to:

- `BaseTest` helper class (https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/BaseTest.cs)
- sample test class that uses `BaseTest` (https://github.com/StarcounterApps/KitchenSink/blob/master/test/KitchenSink.Tests/Test/SectionBoolean/CheckboxPageTest.cs)